src/Indicator.vala

```diff
@@ -84,14 +83,13 @@ public class DateTime.Indicator : Wingpanel.Indicator {
             component_listbox.set_placeholder (placeholder_label);
             component_listbox.set_sort_func (sort_function);
 
-            var scrolled_window = new Gtk.ScrolledWindow (null, null) {
+            var scrolled_window = new Gtk.ScrolledWindow () {
                 hscrollbar_policy = Gtk.PolicyType.NEVER,
                 child = component_listbox
             };
 
-            var settings_button = new Gtk.ModelButton () {
-                text = _("Date & Time Settings…")
-            };
+            var settings_button = new Gtk.Button.with_label (_("Date & Time Settings…"));
```

Comment:
Should be PopoverMenuItem to make sure we set accessible role etc https://github.com/elementary/wingpanel/pull/607